src/ClientRequest.php

```diff
@@ -14,9 +14,11 @@ class ClientRequest extends BasicRequest implements ClientRequestInterface
     function getHeader(bool $public): array
     {
         if ($public) {
-            return [];
+            return $this->headers;
         }
-        return ['Authorization' => "Bearer " . $token = Request::session()->get('pxc.token')];
+        return array_merge(
+            ['Authorization' => "Bearer " . $token = Request::get('pxc.token')], $this->headers
```

Comment:
@QuyenHoang06 try to get the token either for the session or from the request object. We have internal API client that is built in Laravel and this client stores the token in a session. 

@QuyenHoang06 can you verify that pxc.token will work with session and a client's request that has token?

Reply:
Hi @jgardezi , I have updated follow your comment.
Could you please review it?